xdp-dump/xdpdump.c

```diff
@@ -79,6 +81,12 @@ static struct prog_option xdpdump_options[] = {
 		      .short_opt = 'i',
 		      .metavar = "<ifname>",
 		      .help = "Name of interface to capture on"),
+#ifdef HAVE_LIBBPF_PERF_BUFFER__CONSUME
+	DEFINE_OPTION("perf-wakeup", OPT_U32, struct dumpopt, perf_wakeup,
+		      .short_opt = 4,
```

Comment:
Are these non-printable short opts really needed? If so, I'd prefer to fix params.c so they aren't...

Reply:
Yes, they are, else you would crash :( Your current code needs them.

Reply:
Right, okay, let's just merge this as-is, and I'll look into fixing params.c later...

Reply:
Don't forget you need to merge PcapNG first for the tests to work.